content/docs/reference-react.md

```diff
@@ -13,43 +13,43 @@ redirect_from:
   - "docs/top-level-api-zh-CN.html"
 ---
 
-`React` is the entry point to the React library. If you load React from a `<script>` tag, these top-level APIs are available on the `React` global. If you use ES6 with npm, you can write `import React from 'react'`. If you use ES5 with npm, you can write `var React = require('react')`.
+`React` 是 React 函式庫的進入點。 如果你使用 `<script>` 標籤載入 React, 這些頂層 API 可以在 `React` 全域變數使用。 如果你撰寫 ES6 並使用 npm，你可以寫 `import React from 'react'` 。如果你撰寫 ES5 並使用 npm，你可以寫 `var React = require('react')` 。
```

Comment:
```suggestion
`React` 是 React 函式庫的進入點。 如果你使用 `<script>` 標籤載入 React，這些頂層 API 可以在 `React` 全域變數使用。 如果你撰寫 ES6 並使用 npm，你可以寫 `import React from 'react'` 。如果你撰寫 ES5 並使用 npm，你可以寫 `var React = require('react')` 。
```

Reply:
merged both of your suggestions. thanks!